sync_issues_to_jira/sync_issue.py

```diff
@@ -360,6 +362,10 @@ def _get_jira_issue_type(jira, gh_issue):
     issue_types = jira.issue_types()
 
     for gh_label in gh_labels:
+        # Type: Feature Request label should match New Feature issue type in Jira
+        if gh_label == 'Type: Feature Request':
+            print('GitHub label is \'Type: Feature Request\'. Mapping to New Feature Jira issue type')
+            return {"id": 10101}  # JIRA API needs JSON here. 10101 is id for New Feature issue type
```

Comment:
@tom-borcin  is `10101`  stays the same in all cases?

I would suggest extracting it to a constant at the top of the file, like `JIRA_NEW_FEATURE_TYPE_ID=10101`. Magic numbers in functions are always a bit confusing.

Reply:
Good point, fixed now.

Yes, it should stay the same across all projects in our Jira instance.